rosidl_generator_java/resource/action.java.em

```diff
@@ -76,6 +80,41 @@ import @(action_import);
 
 public class @(type_name) implements ActionDefinition {
 
+  public static class SendGoalRequest extends @(type_name)_SendGoal_Request implements GoalRequestDefinition {
+    public List<Byte> getGoalUuid() {
+      return super.getGoalId().getUuid();
+    }
+  }
+
+  public static class SendGoalResponse extends @(type_name)_SendGoal_Response implements GoalResponseDefinition {
+    public void accept(boolean accepted) {
+      super.setAccepted(accepted);
+    }
+
+    public void setStamp(int sec, int nanosec) {
```

Comment:
TBH, I would have used the original signature if I could, but there are two problems:
1. We can't depend on `builtin_interfaces` in `rcljava_common`, to avoid a circular dependency
2. `super.setStamp` returns the object `@(type_name)_SendGoal_Response`